.github/actions/publish/action.yml

```diff
@@ -66,8 +66,8 @@ runs:
         image_and_digest=$(echo "$ARTIFACTS" | jq -r '.[] | select (.type=="Docker Manifest") | .path')
         image=$(echo "${image_and_digest}" | cut -d'@' -f1 | cut -d':' -f1)
         digest=$(echo "${image_and_digest}" | cut -d'@' -f2)
```

Comment:
While we're at it, can you add a `name` to both of these these bash steps? Right now they just show up as "set -euo pipefail" in the logs.